src/subdomains/supporting/payin/services/payin.service.ts

```diff
@@ -96,19 +94,27 @@ export class PayInService {
     await this.payInRepository.save(payIn);
   }
 
-  async updateAmlCheck(payInId: number, amlCheck: CheckStatus): Promise<void> {
-    await this.payInRepository.update(payInId, { amlCheck });
+  async updateForwardConfirmation(payInId: number, forwardConfirmation: boolean): Promise<void> {
+    await this.payInRepository.update(payInId, { isForwardConfirmed: forwardConfirmation });
   }
 
   async returnPayIn(
     payIn: CryptoInput,
     purpose: PayInPurpose,
-    returnAddress: BlockchainAddress,
+    returnAddress: string,
     route: Staking | Sell | Swap,
+    chargebackAmount: number,
   ): Promise<void> {
-    const amlCheck = route.user.userData.kycLevel === KycLevel.REJECTED ? CheckStatus.FAIL : CheckStatus.PASS;
-    payIn.triggerReturn(purpose, returnAddress, route, amlCheck);
+    if (payIn.isForwardConfirmed) throw new BadRequestException('CryptoInput already forwarded');
 
```

Comment:
Is it already forwarded, or just approved for forwarding?

Reply:
Hehe technically it's just approved for forwarding. But I would say in most cases if a payIn has this column true, it is already forwarded. 
Don't know we could, I think we can change it if you want